sense-android-library/src/nl/sense_os/service/provider/LocalStorageSql.java

```diff
@@ -65,6 +65,14 @@ public void onUpgrade(SQLiteDatabase db, int oldVers, int newVers) {
             db.execSQL("DROP TABLE IF EXISTS " + VALUES_TABLE_NAME);
             onCreate(db);
         }
+
+        public SQLiteDatabase getWritableDatabase(){
+          return getWritableDatabase("password");
```

Comment:
i'm not sure either if this part of library is still used or not. Maybe i need to synchronize this with the DBHelper class in nl.sense_os.service.storage?